examples/ghz/ghz_5.yml

```diff
@@ -1,10 +1,12 @@
-amplitudes:
-  - "01000"
-  - "01110"
-  - "10101"
-  - "10001"
-  - "10010"
-  - "11111"
+output:
+  output_method: list
+  bitstrings:
+    - "01000"
+    - "01110"
+    - "10101"
+    - "10001"
+    - "10010"
+    - "11111"
```

Comment:
Think can change `output_method` to `method` as output is already in the level above so is redundant.

Also could be clearer to follow if parameters are put another level down from method. Something like:

```
output:
  method: list
  params:
    bitstrings:
      - "01000"
      - "01110"
      - "10101"
      - "10001"
      - "10010"
      - "11111"
```